redsky/streamer.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+from uuid import uuid4
+import os
+from time import time
+
+doc_name_save_func_mapping = {'start', 'insert_run_start',
```

Comment:
I was thinking of using this later to have a single interface at the end so `blob_func(name, **doc)` where `blob_func` uses this mapping to properly issue the doc.